community-nightlies/deb.sh

```diff
@@ -87,8 +87,8 @@ pgdg_check ()
     apt-get update &> /dev/null
     echo "done."
 
-    if ! apt-cache show postgresql-15 &> /dev/null; then
-      echo "PGDG repositories don't have postgresql-15 package for your operating system"
+    if ! apt-cache show postgresql-16 &> /dev/null; then
+      echo "PGDG repositories don't have postgresql-16 package for your operating system"
```

Comment:
This conditional clause can cause failures in el/7 where postgresql-16 packages are not supported.